closed/autoconf/custom-hook.m4

```diff
@@ -417,7 +417,9 @@ AC_DEFUN([CONFIGURE_OPENSSL],
         if test -s "$OPENSSL_DIR/${LIBRARY_PREFIX}crypto${SHARED_LIBRARY_SUFFIX}" ; then
```

Comment:
Could we replace 417 to 423 with:

if  test “x$BUNDLE_OPENSSL" != x && ! test -s "$OPENSSL_DIR/${LIBRARY_PREFIX}crypto${SHARED_LIBRARY_SUFFIX}"; then
    BUILD_OPENSSL=yes
fi

Reply:
Was not able to do exactly as you stated but have tidied it up a bit.